index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+
+    <link rel="stylesheet" href="css/style.css" />
+    <!-- Responsive Stylesheet -->
+    <link rel="stylesheet" href="css/responsive.css" />
+  </head>
+  <body>
+    <header>
+      <div class="header">
+        <div class="nav-bar">
+          <div class="nav-bar-border">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+        </div>
+        <div class="calendor-logo">
+          <div>
+            <img src="calendar.png" height="20px" width="20px" />
+            <span>Kalendorius</span>
+          </div>
+        </div>
+        <div class="day">
+          <div>Šiandien</div>
```

Comment:
Use `button` tag on anything that is clickable

Reply:
But this div not clickable it just shows with hover what day is today.